build.gradle.kts

```diff
@@ -90,14 +90,20 @@ tasks {
   }
 }
 
-task<Test>("integrationTest") {
```

Comment:
these changes will ensure this task keeps working when upgrading to gradle v9

More info: https://mojdt.slack.com/archives/CTLQ5TCNN/p1755856117469149